backend/docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: '2.2'
+
+services:
+  versewise-cms-backend:
+    build:
+      context: .
```

Comment:
Also it's good to set a container names manually 
- [ ] give your container a name